Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/calendar/CalendarDayViewHolder.kt

```diff
@@ -39,7 +39,7 @@ class CalendarDayViewHolder(v: View) : RecyclerView.ViewHolder(v) {
         textView.contentDescription = day.date.toString(talkBackDateFormat)
 
         // If date is after today - then disable click listener
```

Comment:
Maybe also adjust the comment to reflect the new logic.

Reply:
I just saw your comments, but I have adjusted the comments to reflect the new logic in the latest commit.